### DISCUSSION TITLE
Downloading Analysis Data in Excel

### DISCUSSION THREAD

**Discussion by Zitta109** [sql-bi/Bravo#757] [Q&A]
I've been exploring the forum but haven't come across a similar topic, so I thought I'd bring up a new discussion. I'm looking for a way to download analysis results from Bravo into an Excel format. Specifically, I'm interested in obtaining the table that is generated as the output of the analysis, which includes details like used/unused, Columns, Table, Cardinality, Size, and Weight.

If downloading this data isn't feasible, could someone kindly explain why? Understanding the limitations or restrictions would be very helpful.

To provide some context for my query: I'm currently taking over a substantial number of reports previously managed by junior BI developers. My goal is to enhance and evolve the entire mode. Having the ability to download and thoroughly analyze data from the analyzer, especially to review the usage of repeating columns, would be immensely beneficial for this process.

Looking forward to any insights or advice the community can offer.

Thanks in advance!

**Comment by albertospelta** [comment]
We've got a similar enhancement request already in our backlog, reported in https://github.com/sql-bi/Bravo/issues/505
We're totally on board with making this happen, but as of now, there's no set timeline for when it'll be done.
